src/log/test.cc

```diff
@@ -263,14 +263,14 @@ TEST(Log, TimeFormat)
     ceph::logging::append_time(t, buf, buflen);
     auto c = std::strrchr(buf, '.');
     ASSERT_NE(c, nullptr);
-    ASSERT_EQ(strlen(c + 1), 3);
+    ASSERT_EQ(strlen(c + 1), 3u);
```

Comment:
might want to take this chance to transpose these two arguments. as 
```c++
ASSERT_EQ(expected, actual);
```

Reply:
@tchaikov Aah, I missed it. Thanks for noticing.

Reply:
@tchaikov Done. Please proceed. Thanks.